posts/google-summer-of-code-fellows/gsoc-2024-fellows.qmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "GSoC '24: Developing Tooling for Observational Health Research in Julia"
+project mentor: "Jacob Scott Zelko & Mounika Thakkallapally"
+description: "A brief summary of my project for Google Summer of Code - 2024"
+author: "Jay Sanjay Landge"
+date: "9/5/2024"
+toc: true
+engine: julia
+image: false
+categories:
+  - gsoc
+  - mlj
+  - sql
+  - health-research
+  - data analysis
+---
+
+
+# Hey! Julia Fellows. 👋
+
+# Hi Everyone! 👋
+
+I am Jay Sanjay, currently pursing bachelors degree in Computational Sciences and Engineering at Indian Institute of Technology (IIT) - Hyderabad, India. Coming from a mathematics and data analysis backgroud, I was initially introduced to Julia at my university lecture. Later, I dwewled more into it and same across JuliaHealth community - an intersection of Julia, Health Research, Data Sciences and Informatics. Here, I met some of the great folks in Health Research and Data Informatics, and I decided to take on it in the form of a full fledged summer project. 
+In this blog I will briefly describe about what my project is and what all I did as a part of it.
+
+> If you want to know more about me, you can connect with on [**LinkedIn**](https://www.linkedin.com/in/jay-landge-589439260/) and follow me on [**GitHub**](https://github.com/Jay-sanjay)
+
+
+
+# What is observational health research?
+
+Observational Health Research refers to studies that analyze real-world data (such as patient medical claims, electronic health records, etc.) to understand patient health. These studies often encompass a vast amount of data concerning patient care. An outstanding challenge here is that these datasets can become very complex and grow large enough to require advanced computing methods to process this information.
+
+
+
+# What is Patient Pathways ?
```

Comment:
Could you put these together under one section called like "Background" or something similar?

Reply:
Yeah, exactly I was thinking to give a good synonym for that.